src/InstrumentationEngine.Packages.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.29407.136
+# Visual Studio Version 17
+VisualStudioVersion = 17.2.32207.482
```

Comment:
Ideally we would build CLRIE with the current public stable build of VS. What's the minimum here to support ARM64 targets?

Reply:
(Adding comments here since the other PR won't be merged in) It looks like the earliest supported version for ARM64 is VS 2017 (15.9). We're using toolset v142 and Win SDK 10.0.18362 which are both available for VS 2019 and 2022 so the general public should not be impacted.

The solution file headers here are automatically updated whenever a newer major version of VS is used to build. They don't appear to impact functionality, just the icon used for the .sln file.